simpleclient_jetty_jdk8/src/main/java/io/prometheus/client/jetty/QueuedThreadPoolStatisticsCollector.java

```diff
@@ -37,6 +37,8 @@ public QueuedThreadPoolStatisticsCollector add(QueuedThreadPool queuedThreadPool
         buildGauge("jetty_queued_thread_pool_threads_idle", "Number of idle threads",
             QueuedThreadPool::getIdleThreads),
         buildGauge("jetty_queued_thread_pool_jobs", "Number of total jobs",
+            QueuedThreadPool::getQueueSize),
+        buildGauge("jetty_queued_thread_pool_max", "Max size of thread pool",
```

Comment:
I don't think you've got the right function.

Reply:
I think you're right :man_facepalming: